FirebaseAuth/Tests/Unit/PhoneAuthProviderTests.swift

```diff
@@ -546,24 +545,21 @@
                                     bothClientAndAppID: Bool = false,
                                     reCAPTCHAfallback: Bool = false,
                                     forwardingNotification: Bool = true,
-                                    presenterError: Error? = nil) throws {
+                                    presenterError: Error? = nil) async throws {
       initApp(function, useClientID: useClientID, bothClientAndAppID: bothClientAndAppID,
               testMode: testMode,
               forwardingNotification: forwardingNotification)
       let auth = try XCTUnwrap(PhoneAuthProviderTests.auth)
       let provider = PhoneAuthProvider.provider(auth: auth)
-      let expectation = self.expectation(description: function)
```

Comment:
questions - why can we removed these?

Reply:
@paulb777 mentioned that we don't need expectations in async tests [here](https://github.com/firebase/firebase-ios-sdk/pull/13192#discussion_r1742863161)